types/src/crypto/asymmetric_key.rs

```diff
@@ -314,8 +313,8 @@ pub enum PublicKey {
     #[data_size(skip)] // Manually verified to have no data on the heap.
     Ed25519(ed25519_dalek::PublicKey),
     /// secp256k1 public key.
-    #[data_size(skip)] // Manually verified to have no data on the heap.
-    Secp256k1(k256::PublicKey),
+    #[data_size(skip)]
```

Comment:
We probably don't need this `skip` anymore.

Reply:
Fixed in 899be2413bd24f04ad763b7eafba11fa4b9a33a6